### ISSUE TITLE
Default category for output events

### ISSUE THREAD

**Issue by robertoaloi** [emacs-lsp/dap-mode#491]
According to the [DAP specification](https://microsoft.github.io/debug-adapter-protocol/specification#Events_Output), if a `category` is not specified for an event, `console` should be assumed. In `dap-mode` - unless I'm misusing it -, it seems like `console` output is not rendered anywhere and by default only `stdout` and `stderr` buffers are available (e.g. it is possible to `dap-go-to-output-buffer` for `stdout`.

Would it make sense to render output events with `console` or missing category in the `stdout` buffer? Alternatively, is there a `console` buffer where these events are rendered that I'm missing?

**Comment by yyoncho** [comment]
To me it makes sense to render these. Especially, if vscode is rendering them, have you checked that?

**Comment by robertoaloi** [comment]
@yyoncho Yes, they are rendered in the debug console. You can check the Erlang LS docs about logpoints to see how they behave in VS Code and Emacs:

https://erlang-ls.github.io/articles/tutorial-debugger/#logpoints

For the time being, we are adding the `stdout` category, but `console` should probably do the same.
